### ISSUE TITLE
Bring back support for `{{ j() }}`

### ISSUE THREAD

**Issue by OneMohrTime** [verbb/kint#7]
### What are you trying to do?

In previous versions of Kint for Craft CMS, there was an option to dump directly into the console with `{{ j(variable) }}`. This was really handy when debugging a frontend bug of a previously-designed website quickly, without needing to bash it into the HTML.

As far as I can tell, this is mostly due to functionality changes in `kintphp`, but I don't know further than that.

### What's your proposed solution?

I'm unsure of the implementation, as I don't really write Craft plugins.

I'm getting by using `console.log()` in the HTML, but that takes two steps rather than one.
```twig
{% js %}
    console.log(entry.variable)
{% endjs %}
```

### Additional context

This is briefly mentioned in #5

**Comment by engram-design** [comment]
Yeah, unfortunately there's not much we can do. We've upgraded from Kint 3 to Kint 4, which dropped support for their [kint-js](https://github.com/kint-php/kint-js) package. I'm not a massive fan of rolling that functionality to the plugin, but I'll see how many other people require JS support like this.
